### PR TITLE
Renamed stuff and create guice module for services

### PR DESCRIPTION
**Pull request by danpersa**

### CODE REVIEW

File: src/it/scala/org/zalando/spearheads/innkeeper/AcceptanceSpecsHelper.scala [hunk 1]
```diff
@@ -3,13 +3,13 @@ package org.zalando.spearheads.innkeeper
...
+import akka.http.scaladsl.model.headers.{Authorization, OAuth2BearerToken}
 import akka.stream.ActorMaterializer
-import akka.stream.scaladsl.Source
 import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.time.{Second, Millis, Seconds, Span}
-import slick.backend.DatabasePublisher
+import org.scalatest.time.{Seconds, Span}
+
 import scala.collection.immutable.Seq
+import scala.language.implicitConversions
```
Comment (mo-gr) [inline src/it/scala/org/zalando/spearheads/innkeeper/AcceptanceSpecsHelper.scala]: why is this needed?

**Comment by mo-gr** [comment]
lgtm

File: src/it/scala/org/zalando/spearheads/innkeeper/AcceptanceSpecsHelper.scala (hunk 1 above)
Comment (danpersa) [inline src/it/scala/org/zalando/spearheads/innkeeper/AcceptanceSpecsHelper.scala]: otherwise scala gives a warning... here you explicitly tell the compiler that you want to use implicit conversions...
